pennylane/tape/qnode.py

```diff
@@ -234,9 +246,14 @@ def get_best_method(device, interface):
                 return QNode._validate_backprop_method(device, interface)

```

Comment:
Where would we see rewind sitting here ideally? I feel like it would be nice to benchmark, but potentially before backprop could be cool - making it the default for most devices.

Reply:
I would say:

1. device
2. rewind
3. backprop
4. parameter-shift
5. finite-diff

Reply:
The only issue is we would want `rewind` to support all operations/measurements that backprop supports for it to be above backprop

Reply:
Backprop supports everything doesn't it? :laughing:

Reply:
😆